npm/css-src/sass/overrides/_progress.scss

```diff
@@ -0,0 +1,32 @@
+.progress-display[value] {
+
+    /* Get rid of the default appearance */
+    appearance: none;
+    border: none;
+    /* Dimensions */
+    width: 100%; height: 20px;
+    background-color: rgb(245,245,245);
```

Comment:
The design system comes with a [colour palette](https://frontend.design-system.service.gov.uk/sass-api-reference/#helpers-colour). I think we should use the palette by default so that the progress bar is consistent with the rest of the site.

If we want to use a colour that's not in the palette, I'd suggest extracting the colour to a sass file for our extended palette, so that all the colours can be referenced in one place.

Reply:
Also, if we are using our own colours, it might be good to add them as hex values rather than RGB, again for consistency with the design system: https://github.com/alphagov/govuk-frontend/blob/v3.6.0/src/govuk/settings/_colours-applied.scss#L135

Unless there's a good reason to use RGB/RGBA instead?

Reply:
Thanks for converting the colours to hex values. Did you take a look at the design system's colour palette? Can we use colours from there or define our own?

Reply:
That's all done, I've updated our colours to match the gov uk colour palette and so now they are more consistent to the design standards and look more cleaner too.